cpp/memilio/math/euler.h

```diff
@@ -42,8 +42,8 @@ class EulerIntegratorCore : public IntegratorCore<FP>
      * @param[in,out] dt current time step h=dt
      * @param[out] ytp1 approximated value y(t+1)
```

Comment:
Something like:
* @param The last argument is not used, as the Euler method is not adaptive.
?

Reply:
This would lead to a parameter named "The". But we could name it and mark it as unused in the function body?